students/Dkuchan/ClassProject/PythonClassProject.py

```diff
@@ -0,0 +1,625 @@
+       # self.fieldpos = {matchlist[i].matchnumber: matchlist[i].fieldpos for i in matchlist if matchlist[i].teamnumber == self.number}
+
+    def fillautogears(self):
+        for i in range(0, len(matchlist)):   #I wanted to do this in a list 
+            if matchlist[i].team == self.number:  #did your team play in this match
+                self.autogears.append(int(matchlist[i].autogears))
+       # self.autogears = {matchlist[i].matchnumber: matchlist[i].autogears for i in matchlist if matchlist[i].teamnumber == self.number}
+
+    def fillautokpa(self):
+        for i in range(0, len(matchlist)):   #I wanted to do this in a list 
+            if matchlist[i].team == self.number:  #did your team play in this match
+                self.autokpa.append(int(matchlist[i].autokpa))
+       # self.autokpa = {matchlist[i].matchnumber: matchlist[i].autokpa for i in matchlist if matchlist[i].teamnumber == self.number}
+    
+    def fillclimb(self):
+        for i in range(0, len(matchlist)):   #I wanted to do this in a list 
+            if matchlist[i].team == self.number:  #did your team play in this match
+                self.climb.append(int(matchlist[i].climb))
+       # self.climb = {matchlist[i].matchnumber: matchlist[i].climb for i in matchlist if matchlist[i].teamnumber == self.number}
+
+
+def testfidelityUI():
+    print()
+    print()
+    print("You have chosen to test a matches fidelity.")
+    print("These matches are currently in the database.")
+    showmatchesUI()
+    print("Which match would you like to test?")
+
+
+def testmatchfidelity(matchnumber):
+    # Tests to see if there are 6 teams in a match teamnumber.
+    # Tests to make sure there are no repeats in teams.
+    matchcount = 0
+    teamvar = None
+    teamslist = []
+    for i in range(0, len(matchlist)):
+        if matchlist[i].matchnumber == matchnumber:
+            matchcount+=1
+            teamslist.append(int(matchlist[i].team))
+    print("There are " + len(teamslist) + "/6 logs in the database for match " + matchnumber + ".")
+    for i in teamlist:
+        checkteamnum = teamslist[i]
+        teaminmatch = 0
+        for j in teamlist:
+            if teamlist[j] == checkteamnum:
+                teaminmatch += 1
+        if teaminmatch > 1:
+            print("Team " + str(checkteamnum) + "appears " + teaminmatch + "time(s) in match " + matchnumber + ".")
+        else:
+            print("This match is complete.")
+
+# def updateteams():
+    # This function is intended to refresh team objects with the latest match data.teamnumber
+
+
+def teamcomparisonUI():
+    # this function creates a comparison of 2 teams for quick reference.
+    print()
+    print("Welcome to the Team Comparison Tool")
+    print("Which teams would you like to compare?: ")
+    print()
+
+
+def populateteamdict():
+    # this function creates a dict where the key is
+    # the team number (string) and the value is the team object.
```

Comment:
I would use either a number or a string for a team id -- if it's a number, use an int. or just let it be an arbitrary string.